templates/web/base/admin/body_translation_edit.html

```diff
@@ -0,0 +1,31 @@
+[% INCLUDE 'admin/header.html' title=tprintf(loc('Edit %s Translation for %s'), language_name, body.name ) -%]
+
+<form method="post" action="[% c.uri_for('body', body_id ) %]" enctype="application/x-www-form-urlencoded" accept-charset="utf-8" id="category_edit">
+    <input type="hidden" name="translation_id" value="[% translation.id| html %]" >
+    <input type="hidden" name="token" value="[% csrf_token %]" >
+    <input type="hidden" name="posted" value="edit_translation" >
+    <input type="hidden" name="language" value="[% translation.lang| html %]" >
+    <p>
+    <strong>Original</strong>: [% body.name | html %]
```

Comment:
I wanted to make it totally obvious what you were translating. It's a bit redundant but having the thing you are translating next to the translation and also not embedded in a title seemed a bit clearer.